r/R/record-batch-reader.R

```diff
@@ -104,6 +105,16 @@ RecordBatchReader <- R6Class("RecordBatchReader",
     schema = function() RecordBatchReader__schema(self)
   )
 )
+RecordBatchReader$create <- function(..., batches = list(...), schema = NULL) {
```

Comment:
I think I just learned something new about R. `batches` here is the equivalent of Python's `*args`, right?

Reply:
In effect, yes, or `**kwargs`. This is a convenient pattern when you want to accept `...` to turn into a list but may have already assembled the list.